### ISSUE TITLE
Incompatibility with Spectrum mod

### ISSUE THREAD

**Issue by BrekiTomasson** [cybercat5555/Miskatonic-Mysteries-Fabric#54]
Trying to run one of the latest builds from Actions together with the rest of the mods in my modpack, and discovered an incompatibility with the mod `spectrum` - more specifically, with the `bannerpp` mod/library that it includes. See below from the logs:

```
[02:31:41] [main/WARN]: Method overwrite conflict for setTranslation in flywheel.fabric.mixins.json:Matrix4fMixin from mod flywheel, previously written by io.github.fabricators_of_create.porting_lib.mixin.common.Matrix4fMixin. Skipping method.
[02:31:41] [main/WARN]: @Redirect conflict. Skipping bannerpp.mixins.json:LoomContainerMixin from mod bannerpp->@Redirect::addBppLoomPatternCondition(Lnet/minecraft/class_3915;)I with priority 1000, already redirected by mixins.miskatonicmysteries.common.json:block.LoomContainerMixin from mod miskatonicmysteries->@Redirect::mm$addBppLoomPatternCondition(Lnet/minecraft/class_3915;)I with priority 1001
[02:31:41] [main/WARN]: @Redirect conflict. Skipping bannerpp.mixins.json:LoomContainerMixin from mod bannerpp->@Redirect::addBppLoomPatternCondition(Lnet/minecraft/class_3915;)I with priority 1000, already redirected by mixins.miskatonicmysteries.common.json:block.LoomContainerMixin from mod miskatonicmysteries->@Redirect::mm$addBppLoomPatternCondition(Lnet/minecraft/class_3915;)I with priority 1001
[02:31:41] [main/WARN]: @ModifyConstant conflict. Skipping bannerpp.mixins.json:LoomContainerMixin from mod bannerpp->@ModifyConstant::disarmVanillaPatternLimitCheck(I)I with priority 1000, already redirected by mixins.miskatonicmysteries.common.json:block.LoomContainerMixin from mod miskatonicmysteries->@ModifyConstant::mm$disarmVanillaPatternLimitCheck(I)I with priority 1001
```

Due to this, core functionalities in `spectrum` fail to work as intended and the game crashes before launch.

`Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector addBppLoomPatternCondition(Lnet/minecraft/class_3915;)I in bannerpp.mixins.json:LoomContainerMixin from mod bannerpp failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap bannerpp-refmap.json`

Disabling either `spectrum` or `miskatonicmysteries` makes everything start up and work as intended.

**Comment by BrekiTomasson** [comment]
Any news on this? It's still an issue in the released v0.1.0.

**Comment by Aelpecyem** [comment]
I'll look into it later this week
